### DISCUSSION TITLE
What is the correct Spatial Transcriptomics data input?

### DISCUSSION THREAD

**Discussion by schmerlain14** [satijalab/seurat#4808] [Q&A]
We have a very similar data set that you have, 2 individuals with each two conditions. While doing the tutorial I recognized that you work with the count data not the aggregated and that you only merge 2 datasets instead of all 4. Why is that so? Why don't we use the aggregated data? What is the correct way to do it then, loading the 4 count data sets and then merge them each by each/two and two?

And also shouldn't the SCTransform be done for the merged data too?

I am talking about this stuff:
```
InstallData("stxBrain")
brain <- LoadData("stxBrain", type = "anterior1")
brain <- SCTransform(brain, assay = "Spatial", verbose = FALSE)
## Working with multiple slices in Seurat
brain2 <- LoadData("stxBrain", type = "posterior1")
brain2 <- SCTransform(brain2, assay = "Spatial", verbose = FALSE)
brain.merge <- merge(brain, brain2)

```

**Comment by saketkc** [comment]
The tutorial only works with 2 datsets as an example. Regarding SCTransform, I have answered a related question here: https://github.com/satijalab/seurat/issues/5009#issuecomment-907426252

**Reply by schmerlain14** [reply]
Thank you for the answer.

There is still an open question now, does it make sense to integrate such data? And how can I add the patient and condition information to the anchor creation?
